### PR TITLE
added memory fallback for titanium android

### PR DESCRIPTION
**Pull request by edewit**
@wei-lee need to add this on android as it doesn't have a window name

### CODE REVIEW

File: src/modules/data.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ var data = {
   //dom adapter doens't work on windows phone, so don't specify the adapter if the dom one failed
   //we specify the order of lawnchair adapters to use, lawnchair will find the right one to use, to keep backward compatibility, keep the order
   //as dom, webkit-sqlite, localFileStorage, window-name
-  DEFAULT_ADAPTERS : ["dom", "webkit-sqlite", "window-name"],
+  DEFAULT_ADAPTERS : ["dom", "webkit-sqlite", "window-name", "memory"],
```
Comment (wei-lee) [inline src/modules/data.js]: but why dom & wekit-sqlite doesn't work? Is this change for Titanium?

**Comment by edewit** [comment]
@wei-lee yes this is for titanium, well dom is obvious as it's a native front-end and 'maybe' that is why there is no webkit-sql as well as it's only a javascript runtime

**Comment by wei-lee** [comment]
@edewit ok, but if we use memory, all the data will not be persisted for Titanium. Is this ok?

**Comment by edewit** [comment]
guess it was no different if you are using 'window-name', I'm not sure what is persisted with Lawn chair.

**Comment by edewit** [comment]
@wei-lee so good to merge?

**Comment by wei-lee** [comment]
I think perhaps better use this? https://github.com/feedhenry/fh-js-sdk/blob/master/libs/lawnchair/lawnchairTitanium.js. Turns out we already have a data adapter for Titanium!

**Comment by edewit** [comment]
@wei-lee good to merge?

**Comment by nialldonnellyfh** [comment]
@edewit Looks like it needs a rebase from master

**Comment by edewit** [comment]
@nialldonnellyfh rebased

**Comment by nialldonnellyfh** [comment]
@edewit It needs a version bump.

**Comment by edewit** [comment]
@wei-lee @nialldonnellyfh it needs a rebase again, can't we merge this one?

**Comment by matzew** [comment]
I agree w/ @edewit, lack of frequent merges cause pain to maintain PRs

ship it :smile:

On Wednesday, 18 May 2016, Erik Jan de Wit <notifications@github.com> wrote:

> @wei-lee <https://github.com/wei-lee> @nialldonnellyfh
> <https://github.com/nialldonnellyfh> it needs a rebase again, can't we
> merge this one?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/feedhenry/fh-js-sdk/pull/161#issuecomment-219973694>
>


-- 
Sent from Gmail Mobile

**Comment by wei-lee** [comment]
LGTM. merge it

**Pull request merged**